day_two_guide/topics/scaling_etcd.adoc

```diff
@@ -0,0 +1,432 @@
+////
+Scaling etcd
+
+Module included in the following assemblies:
+
+* day_two_guide/host_level_tasks.adoc
+////
+
+Scaling the etcd cluster can be performed vertically by adding more resources to
+the etcd hosts, or horizontally by adding more etcd hosts.
```

Comment:
s/Scaling the etcd cluster can be performed/You can scale the etcd cluster

no comma